design-documents/graph-ql/coverage/store-currency-symbol.md

```diff
@@ -0,0 +1,24 @@
+# Coverage for currency symbol
+
+Magento allows to use custom currency symbol.The current graphql implementation does not fully support custom currency symbol feature. The currency query only returns base currency symbol and current store default currency symbol.
+ `available_currency_codes` should be deprecated and instead `available_currency` field should be added, or both should be kept
+ 
+ ## Schema
+```graphql
+type CurrencyData {
```

Comment:
this is ok but instead of using `CurrencyData` - the word `data` is ambiguous
I can't think of a better name than Currency but it's taken.
Maybe we want to rename existing type `Currency` to CurrencySettings  
currency: Currency deprecating this - 
add currencySettings: CurrencySettings

we can name CurrencyData to CurrencyObject then in the future rename it back to Currency {
    currency_code: String
    currency_symbol: String

}

Reply:
Thanks @cpartica . I have updated the schema